app/Http/Controllers/FavoritesController.php

```diff
@@ -14,13 +14,13 @@ public function store(Request $request, Session $session): void
     {
         $cityId = $request->input("city_id");
 
-        $existingFavorite = Favorites::where("user_id", request()->user()->id)
+        $existingFavorite = Favorites::where("user_id", request()->user()?->id)
```

Comment:
1. use `$request` variable, not helper function.
2. move user id to some variable and then use it

Reply:
Done